CHANGELOG.md

```diff
@@ -1,14 +1,17 @@
 # TAMR AWS EMR module
+## v2.0.0 - April 7th 2021
```

Comment:
Nit: I don't think this is a breaking change. Maybe would make more sense to bump to 1.1.0

Reply:
makes sense- `1.1.0` should be okay

Reply:
Yeah, `1.1.0` seems right.

The guideline for versioning is:
major - breaking change
minor - new feature
patch - bug fix